modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -41,107 +45,112 @@
   $backup_enabled = false
 ) {
 
-  validate_re($version, '^\d+\.\d+\.\d+$', 'govuk_elasticsearch::version must be in the form x.y.z')
-  include augeas
-  validate_bool($manage_repo)
+  if $enabled {
```

Comment:
Yep, pretty much.  With the absence of a "roles & profiles" design pattern we have to gate classes in this way.  Sure, I'll move te gate up.